app/bundles/CoreBundle/Assets/json/countries.json

```diff
@@ -31,13 +38,14 @@
     "Cambodia",
     "Cameroon",
     "Canada",
+    "Cayman Islands",
     "Central African Republic",
     "Chad",
     "Chile",
     "China",
     "Colombia",
     "Comoros",
-    "Republic of the Congo",
```

Comment:
https://en.wikipedia.org/wiki/ISO_3166-1 lists the following two states with "Congo" in their name:

   * Congo
   * Congo (Democratic Republic of the)

Is there a reason why we've removed any reference to either here?